### DISCUSSION TITLE
Suggestion for unused Invites

### DISCUSSION THREAD

**Discussion by CYB3R1NS4N3** [idena-network/idena-desktop#786] [Ideas]
My suggestion is to get an idena coins for unused invitations. Sometimes you really don't have anyone to share an invite with, maybe it would be good to get an idena coins for unused invitations since you cannot use them for future validations.

**Comment by midenaio** [comment]
@CYB3R1NS4N3 In the previous versions of Idena protocol there were lottery-based rewards for non-spent invitations to prevent sporadic invitations. However it did not work well and was removed from the protocol. Please check more details here:
https://medium.com/idena/idena-2021-community-report-c7b2b1b262dc

**Comment by andrei0x309** [comment]
@midenaio Has the `spoil invitation system` been impactful? Maybe there are some stats, but I find it hard to believe that is used much, I never saw an invitation shared publicly, not saying it isn't a thing but it sure looks like a rarity. 

One thing I don't understand is why give higher rewards for invitations if they are given earlier, I think would be better if it was the opposite, the higher the reward the closer you are to the validation ceremony. Because we all know that the identity will have a higher chance of success if is closer to the period of the validation process.

What I think would be a cool system, is if you would give 1/6 of the reward for an unspent invitation, and combine that with giving higher rewards if the invitation is activated closer to the validation.

This would be great because it should incentives people to find quality members in order to get better rewards and also improve the number of validated identities while also providing a low usage for unspent invitations.
